.rubocop_todo.yml

```diff
@@ -0,0 +1,27 @@
+# ruby 1.8.7 required trailing dots
+Layout/DotPosition:
+  EnforcedStyle: trailing
+
+# Over time we'd like to get this down, but this is what we're at now.
+Metrics/CyclomaticComplexity:
+  Max: 9 # default: 6
+
+# Over time we'd like to get this down, but this is what we're at now.
+Metrics/LineLength:
+  Max: 186 # default: 80
+
+# Over time we'd like to get this down, but this is what we're at now.
+Metrics/MethodLength:
+  Max: 43 # default: 10
+
+# each_with_object was unavailable on ruby 1.8.7
+Style/EachWithObject:
+  Enabled: false
+
+# ruby 1.8.7 only had support for hash rockets
+Style/HashSyntax:
+  EnforcedStyle: hash_rockets # default: ruby19
+
+# the new lambda syntax was not supported in ruby 1.8.7
+Style/Lambda:
+  Enabled: false
```

Comment:
I'm going to assume the intent is to remove all these at some point?

Reply:
Yep, extracted those with promises to improve things, or 1.8 support excuses.